File: benchmarks/python/array_ops.py
import time

SIZE = 1000

start_time = time.time()

# Create and populate array
arr = []
for i in range(SIZE):
    arr.append(i * 2)

# Sum all elements
total = sum(arr)

end_time = time.time()

print(f"Array size: {SIZE}")
print(f"Sum: {total}")
print(f"Time: {(end_time - start_time) * 1000:.2f}ms")

File: benchmarks/ruby/array_ops.rb
SIZE = 1000

start_time = Time.now

# Create and populate array
arr = []
(0...SIZE).each do |i|
    arr << i * 2
end

# Sum all elements
total = arr.sum

end_time = Time.now

puts "Array size: #{SIZE}"
puts "Sum: #{total}"
puts "Time: #{((end_time - start_time) * 1000).round(2)}ms"